packages/paste-core/components/corner-ornament/src/CornerOrnamentContext.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import { CornerOrnamentContextInterface } from "./types";
+
+export const CornerOrnamentContext = React.createContext<CornerOrnamentContextInterface | null>(null);
+
+export const useCornerOrnamentContext = (): CornerOrnamentContextInterface => {
+  const context = React.useContext(CornerOrnamentContext);
+  if (!context) {
+    throw new Error("useCornerOrnamentContext must be used with CornerOrnamentContextProvider");
```

Comment:
question: who is this error for?

Reply:
todo: remove error

Reply:
Kept error in but with mentioned it should be used within within container. Without this check it throws type errors and feels safer.